privaterelay/templates/includes/domain-alias-dashboard.html

```diff
@@ -2,134 +2,19 @@
-              </div>
-              <div class="relay-email-column alias-stats-wrapper">
-                <div class="nested-grid flx flx-row hide-mobile">
-                  {% include "includes/alias-stats.html" with relay_address=domain_address only %}
-                </div>
-              </div>
-            </div><!-- end row -->
-            <div class="relay-email-row card-row details-wrapper flx">
-              <div class="show-mobile nested-grid flx flx-row">
-                  {% include "includes/alias-stats.html" with relay_address=domain_address only %}
-              </div>
-              <div class="details-row">
-                <section class="relay-email-column details-column flx flx-col">
-                  <span class="card-small-text created-label">First emailed:</span>
-                  <span class="email-value card-small-text text-dark">{{ domain_address.first_emailed_at|date:"M j, Y" }}</span>
-                </section>
-
-                <!-- forwarding message -->
-                <section class="relay-email-column hide-mobile details-column flx flx-col">
-                  <span class="relay-email-label email-forwarding-label card-small-text">Forward emails to</span>
-                  <span class="text-dark demi primary-email-address card-small-text">{{ request.user.email }}</span>
-                </section>
-              </div>
-
-
-              <!-- delete email address -->
-              <section class="relay-email-column column-delete center-value">
-                <form action="{% url 'emails-index' %}" method="POST" class="delete-email-form flx jst-cntr al-cntr" data-delete-relay="{{ domain_address.address }}@{% user_email_domain user_profile %}">
-                  <input type="hidden" name="method_override" value="DELETE">
-                  <input type="hidden" name="api_token" value="{{ user_profile.api_token }}">
-                  <input type="hidden" name="domain_address_id" value="{{ domain_address.id }}">
-                  <button type="submit" name="delete" value="" class="dashboard-delete-button delete-relay-email-button" aria-label="Delete this alias">
-                    <span class="remove-label relay-email-label card-small-text">Delete</span>
-                  </button>
-                </form>
-              </section> <!--end delete-->
-
-            </div><!--row-end-->
-          </div> <!-- end card-->
-        </div> <!-- end relay-email-->
-        {% endfor %}
+{% if not user_profile.has_unlimited %}
+    <div class="dashboard-mpp-promo">
+      <div class="dashboard-mpp-promo-container inset">
+        <h3>
+          {% ftlmsg 'banner-pack-upgrade-headline-html' %}
+        </h3>
+        <p>
+          {% ftlmsg 'banner-pack-upgrade-copy' %}
+        </p>
+        <a class="btn btn--blue" target="_blank" rel="noopener noreferrer"
+         href="https://accounts.stage.mozaws.net/subscriptions/products/{{ settings.PREMIUM_PROD_ID }}?plan={{ settings.PREMIUM_PRICE_ID }}">
```

Comment:
Hard-coded `accounts.stage.mozaws.net` needs to be replaced with a `settings` value from an environment variable.

Reply:
Fixing.

Reply:
Fixed!

Reply:
I have the URL fall back to prod if no local ENV is set and added it to the env-dist.